test/inductor/test_cudagraph_trees.py

```diff
@@ -269,6 +269,19 @@ def foo(x, y):
                 captured_output[0]
             )
 
+        @torch._inductor.config.patch("triton.skip_dynamic_graphs", True)
```

Comment:
Should you also test that multiple compiles with different shapes still return correct results? or is that the status quo

Reply:
already tested: https://github.com/pytorch/pytorch/commit/10cbc9a063d20116ded39fb54a59db9d17e53acf#diff-70ac3047d6e27843ff4f81ee908669839ca9551814ccbd47b2c3354997f87c73R1123-R1180